modules/xray/shared/src/main/scala/natchez/xray/XRaySpan.scala

```diff
@@ -7,10 +7,10 @@ package natchez.xray
 import cats.effect._
 import cats.syntax.all._
 import cats.effect.Resource.ExitCase
-
 import natchez._
 import natchez.TraceValue._
 import cats.effect.Resource
+
 import java.net.URI
 import java.time.Instant
```

Comment:
If possible, `FiniteDuration` > `Instant` for JS support.

Edit: oh, is this baked into Natchez?

Reply:
No, it was possible to make the switch to `FiniteDuration`. In doing so, I noticed that there were several other places where I was able to relax the effect constraint from `Sync` to something less powerful, so double win!

Reply:
Nice!